usagov/usagov-2021
# USAGOV-587-Allow-content-admins-to-edit-images

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-587

## Description

Give Content Admins access to the Media Library so they can get to the media editing pages.


## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



- Log in to Drupal as a content administrator 
  - You may need to add a user and create a new content administrator to log in as
- [ ] Confirm that, as a content admin, you can go to Content > Media > Media Library
- Choose an image and click the edit icon
- [ ] Confirm that you can edit and save the Alternative text field for that image

### Requires New Config

### Requires New Content

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
